organization/_posts/2021/2021-09-02-coordination.md

```diff
@@ -0,0 +1,149 @@
+
+- Discussion about the success rate noted that:
+  - Our success rate has steadily improved over the years
+  - It's usually higher than the program average
+
+An EP newsletter article has been prepared by Graeme and Andrei.
+
+## Working Group Updates
+
+### Data Analysis
+
+- Starting to plan next topical meetings; more details to follow.
+
+### Detector Simulation
+
+- Starting to prepare Autumn meetings, contacting speakers.
+- As always, contact the convenors if you have anything you'd like to present or see covered.
+
+### Reconstruction and Software Triggers
+
+- Planning second topical meeting on 4D timing on September 22nd or 29th (tbc)
+- Possibly plan a mini-workshop on heterogeneous architectures in November
+- Getting in touch with people in the field to get a feeling for need / interest of specific topics for discussions at such a workshop
+
+### PyHEP
+
+- Re-start of topical meetings on October 6th.
+
+### Software Training
+
+- Organised the first [virtual Software Carpentry workshop](https://indico.cern.ch/event/1058873/) 25-27 Aug 2021 (75 participants registered)
+  - Software Carpentry workshops (SCW) will be organized every ~3 months.
+
+- First publication on Software training (based on vCHEP 2021 contribution) submitted (1 Aug 2021) to CSBS  - Springer (Computing and Software for Big Science) [arXiv version](https://arxiv.org/abs/2103.00659))
+
+#### 3rd HEP C++ Course and Hands-on Training
+
+- [Event](https://indico.cern.ch/event/1019089/) is running this week. 75 students attending. Another 90 on the waiting list. With this course we will have trained 200 students since October 2020.
+- Many thanks to all lecturers and mentors!!!
+- Meeting notes that we should improve the connection between the C++ teaching and the WG.
+
+### Event Generators
+
+- Not much to report. Very little action in August.
+  - Main focus now is on implementing comments received on the LHCC document and recirculating to the WG.
+  - "Taming the accuracy of event generators" workshop last week, will be interesting for us to look at discussions: <https://indico.cern.ch/event/999271/>
+
+### Frameworks
+
+- Will (hopefully) have a meeting next week after a longer hiatus. (<https://indico.cern.ch/event/1072798/>) With the intention of going back to regular meetings in the autumn once again.
+
+---
+
+## AOB
+
+### HSF Domain Name
+
+Ticket still open, no recent movement.
+
+### SIDIS COST Proposal
+
+We are currently working on a funding proposal (EU COST) in the context of the "Software Institute for Data Intensive Sciences", aiming to bridge the gap between computer scientists and natural science software engineers. The "grand theme" of the proposal will be around the topic of "software sustainability".
+
+For a success proposal it is useful to include participants also from "inclusive target" countries (mostly eastern European countries). In case you have contacts in this direction please get in contact with Graeme or Stefan.
```

Comment:
```suggestion
For a successul proposal it is useful to include participants also from "inclusive target" countries (mostly eastern European countries). In case you have contacts in this direction please get in contact with Graeme or Stefan.
```

Reply:
*successful*